blog/2019-01-28-debezium-0-9-0-cr1-released.adoc

```diff
@@ -0,0 +1,103 @@
+= Debezium 0.9.0.CR1 Released
+gmorling
+:awestruct-tags: [ releases, mysql, mongodb, postgres, sqlserver, oracle, docker ]
+:awestruct-layout: blog-post
+
+Reaching the home stretch towards Debezium 0.9, it's with great pleasure that I'm announcing the first release of Debezium in 2019, *0.9.0.CR1*!
+
+For this release we've mainly focused on sorting out remaining issues in the Debezium link:/docs/connectors/sqlserver/[connector for SQL Server];
+the connector comes with greatly improved performance and has received a fair number of bug fixes.
+
+Other changes include a new interface for event handlers of Debezium's link:/docs/embedded/[embedded engine],
+which allows for bulk handling of change events, an option to export the scale of numeric columns as schema parameter
+as well as a wide range of bug fixes for the Debezium connectors for MySQL, Postgres and Oracle.
+
+== SQL Server Connector Improvements
+
+The SQL Server connector supports blacklisting of specific columns now (https://issues.jboss.org/browse/DBZ-1067[DBZ-1067]).
+That's useful in cases where you'd like to exclude specific columns from emitted change data messages e.g. to data protection considerations.
+
+The "snapshot locking mode" option has been reworked (https://issues.jboss.org/browse/DBZ-947[DBZ-947]) and is named "snapshot isolation mode" now,
+better reflecting its semantics.
+A new mode "repeatable_read" has been added, and "none" has been renamed to "read_uncommitted".
+Please see the connector documentation and the link:/docs/releases/#breaking_changes[migration notes] for more details.
+
+The connector allows for a much higher through-put now, thanks to caching of timestamps for the same LSN (https://issues.jboss.org/browse/DBZ-1078[DBZ-1078]).
+Please refer to the link:/docs/releases/#release-0-9-0-cr1[change log] for details on bugs fixed in this connector.
```

Comment:
Is this link expected to work after the release is done or something is merged?

Reply:
It will be resolved properly in the rendered HTML; it's relative to the root URL, i.e. it'll be rendered as https://debezium.io/docs/releases/#release-0-9-0-cr1.